### PR TITLE
[Request][Serverless][8.16] Visualizations in alert flyout - technical preview + advanced setting

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5878

ESS:
- [Configure advanced settings | Access the event analyzer and session view from the event or alert details flyout](https://security-docs_bk_5963.docs-preview.app.elstc.co/guide/en/security/master/advanced-settings.html#visualizations-in-flyout): Added docs for the `securitySolution:enableVisualizationsInFlyout` advanced setting.
- [View alert details | Visualizations | Expanded visualizations view](https://security-docs_bk_5963.docs-preview.app.elstc.co/guide/en/security/master/view-alert-details.html#expanded-visualizations-view): Added a new sub-section for the **Visualize** tab.
- [Visualize event analyzer | Find events to analyze](https://security-docs_bk_5963.docs-preview.app.elstc.co/guide/en/security/master/visual-event-analyzer.html#find-events-analyze): Made the language more generic and included a tip about the enabling the `securitySolution:enableVisualizationsInFlyout` advanced setting.

Serverless:
- [Configure advanced settings | Access the event analyzer and session view from the event or alert details flyout](https://elastic-dot-co-docs-production-iuohchaqp-elastic-dev.vercel.app/current/serverless/security/advanced-settings#access-the-event-analyzer-and-session-view-from-the-event-or-alert-details-flyout): Same changes as described above.
- [View alert details | Visualizations | Expanded visualizations view](https://elastic-dot-co-docs-production-iuohchaqp-elastic-dev.vercel.app/current/serverless/security/view-alert-details#expanded-visualizations-view): Ditto
- [Visualize event analyzer | Find events to analyze](https://elastic-dot-co-docs-production-iuohchaqp-elastic-dev.vercel.app/current/serverless/security/visual-event-analyzer#find-events-to-analyze): Ditto

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-6cyajfgdt-elastic-dev.vercel.app

* Built with commit 42ea290a8b5311231b091883b853f46c7693164c

> Issues? Visit #next-docs in Slack

**Review by christineweng — DISMISSED** [review]
LGTM! Thank you!

File: docs/detections/alerts-view-details.asciidoc [hunk 1]
```diff
@@ -124,10 +124,32 @@ image::images/visualizations-section-rp.png[Visualizations section of the Overvi
...
-* **Session view preview**: Shows a preview of <<session-view,session view>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
+* **Session view preview**: Shows a preview of <<session-view,Session View>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
 
 * **Analyzer preview**: Shows a preview of the <<visual-event-analyzer,visual analyzer graph>>. The preview displays up to three levels of the analyzed event's ancestors and up to three levels of the event's descendants and children. The ellipses symbol (**`...`**) indicates the event has more ancestors and descendants to examine. Click **Analyzer preview** to open the **Event Analyzer** tab in Timeline.
 
+[discrete]
+[[expanded-visualizations-view]]
+=== Expanded visualizations view
+
+preview:[]
```
Comment (natasha-moore-elastic) [inline docs/detections/alerts-view-details.asciidoc:135]: This ⬆️ renders as the inline `[preview]` label, but the full banner might be more appropriate.

```suggestion
preview::[]
```

File: docs/detections/alerts-view-details.asciidoc [hunk 2]
```diff
@@ -124,10 +124,32 @@ image::images/visualizations-section-rp.png[Visualizations section of the Overvi
 
 Click **Visualizations** to display the following previews:
 
-* **Session view preview**: Shows a preview of <<session-view,session view>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
+* **Session view preview**: Shows a preview of <<session-view,Session View>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline.
```
Comment (natasha-moore-elastic) [inline docs/detections/alerts-view-details.asciidoc:127]: Should this say **Session viewer preview**?

```suggestion
* **Session viewer preview**: Shows a preview of <<session-view,Session View>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
```

File: docs/detections/alerts-view-details.asciidoc [hunk 3]
```diff
@@ -124,10 +124,32 @@ image::images/visualizations-section-rp.png[Visualizations section of the Overvi
...
+
+preview:[]
+
+.Requirements
+[sidebar]
+--
+To use the **Visualize** tab, you must turn on the `securitySolution:enableVisualizationsInFlyout` <<visualizations-in-flyout,advanced setting>>. 
+--
+
+The **Visualize** tab allows you to maintain the context of the Alerts table, while providing a more detailed view of alerts that you're investigating in the event analyzer or Session View. To open the tab, click **Session view preview** or **Analyzer preview** from the right panel.
```
Comment (natasha-moore-elastic) [inline docs/detections/alerts-view-details.asciidoc:143]: Same question about **Session view/viewer preview**

File: docs/detections/visual-event-analyzer.asciidoc [hunk 4]
```diff
@@ -29,7 +29,9 @@ Or
 +
 ** `agent.type:"winlogbeat" and event.module: "sysmon" and process.entity_id : *`
 
-. Events that can be visually analyzed are denoted by a cubical **Analyze event** icon. Select this option to open the event in the visual analyzer. Alternatively, open the alert details flyout, go to the Visualizations section, then click **Analyzer preview**. This opens the **Analyzer** tab in Timeline.
+. Events that can be visually analyzed are denoted by a cubical **Analyze event** icon. Select this option to open the event in the visual analyzer. The event analyzer is accessible from the Hosts, Alerts, and Timelines pages, as well as the alert details flyout.
```
Comment (natasha-moore-elastic) [inline docs/detections/visual-event-analyzer.asciidoc:32]:
```suggestion
. Events that can be visually analyzed are denoted by a cubical **Analyze event** icon. Select this option to open the event in the visual analyzer. The event analyzer is accessible from the **Hosts**, **Alerts**, and **Timelines** pages, as well as the alert details flyout. 
```

File: docs/getting-started/advanced-setting.asciidoc [hunk 5]
```diff
@@ -113,6 +113,14 @@ The `securitySolution:enableAssetCriticality` setting determines whether asset c
 
 Including data from cold and frozen {ref}/data-tiers.html[data tiers] in <<visual-event-analyzer, visual event analyzer>> queries may result in performance degradation. The `securitySolution:excludeColdAndFrozenTiersInAnalyzer` setting allows you to exclude this data from analyzer queries. This setting is turned off by default.
 
+[discrete]
+[[visualizations-in-flyout]]
+== Access the event analyzer and session view from the event or alert details flyout
+
+preview:[]
```
Comment (natasha-moore-elastic) [inline docs/getting-started/advanced-setting.asciidoc:120]:
```suggestion
preview::[]
```

File: docs/getting-started/advanced-setting.asciidoc [hunk 6]
```diff
@@ -113,6 +113,14 @@ The `securitySolution:enableAssetCriticality` setting determines whether asset c
 
 Including data from cold and frozen {ref}/data-tiers.html[data tiers] in <<visual-event-analyzer, visual event analyzer>> queries may result in performance degradation. The `securitySolution:excludeColdAndFrozenTiersInAnalyzer` setting allows you to exclude this data from analyzer queries. This setting is turned off by default.
 
+[discrete]
+[[visualizations-in-flyout]]
+== Access the event analyzer and session view from the event or alert details flyout
```
Comment (natasha-moore-elastic) [inline docs/getting-started/advanced-setting.asciidoc:118]:
```suggestion
== Access the event analyzer and Session View from the event or alert details flyout
```

**Review by natasha-moore-elastic — COMMENTED** [review]
A few minor suggestions, otherwise looks good!

File: docs/detections/alerts-view-details.asciidoc (hunk 2 above)
Comment (nastasha-solomon) [reply docs/detections/alerts-view-details.asciidoc:127]: yeah, it should be "viewer" not "view" here - good catch!

File: docs/detections/alerts-view-details.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/detections/alerts-view-details.asciidoc:143]:
```suggestion
The **Visualize** tab allows you to maintain the context of the Alerts table, while providing a more detailed view of alerts that you're investigating in the event analyzer or Session View. To open the tab, click **Session viewer preview** or **Analyzer preview** from the right panel. 
```

**Review by natasha-moore-elastic — APPROVED** [review]
🚢 🚢 🚢

**Comment by nastasha-solomon** [comment]
Verified that the advanced setting and tab are available in Serverless prod and work as expected.

**Pull request merged**